layers/+spacemacs/spacemacs-defaults/funcs.el

```diff
@@ -570,6 +570,11 @@ FILENAME is deleted using `spacemacs/delete-file' function.."
   (funcall-interactively #'spacemacs/delete-file filename t))
 
 ;; from magnars
+(defcustom spacemacs-delete-current-buffer-file-arbitrary nil
+  "User deletes current buffer and file without confirmation."
+  :type 'boolean
+  :group 'spacemacs)
```

Comment:
Please change the name to "spacemacs-keep-legacy-current-buffer-delete-bindings"

Please change the description so that the key binding change is described and which keys are set when legacy is used and which are set the other way around.

Default for this var should be nil for we emphasize the new binding which brings more functionality while we still allow to keep the old bindings.